wisdem/inputs/analysis_schema.yaml

```diff
@@ -789,93 +821,44 @@ properties:
 
     driver:
         type: object
+        description: Specification of the optimization driver (optimization algorithm) parameters
```

Comment:
Looks like you are undoing Nikhar's changes with yours though.  I think you need to redo that merge in some way.  Let me know if I can help.

Reply:
Oh that's wrong indeed! Let me try to see what's going on